package.json

```diff
@@ -6,6 +6,7 @@
     "@material-ui/core": "^3.0.2",
     "@material-ui/icons": "^3.0.1",
     "cldr-data": "^33.0.0",
+    "create-react-class": "^15.6.3",
```

Comment:
We do not need this and will not need it in the future, so I suggest we remove create-react-class here

Reply:
Alright, it should be good to go now!